### PR TITLE
Add Korean translations

### PR DESCRIPTION
**Pull request by astralkr**
korean translation

**When merged this pull request will:**
- Lots of Korean translation will add

### CODE REVIEW

File: addons/common/stringtable.xml [hunk 1]
```diff
@@ -467,6 +512,7 @@
             <Polish>Wybierz preferowany arsenał używany przez Zeus Enhanced.\nJeśli arsenał ACE nie będzie dostępny to uruchomi się podstawowy wirtualny arsenał BI.</Polish>
             <Russian>Выбор предпочитаемого Арсенала\nlfЕсли не установлен ACE, будет запущен стандартный Виртуальный Арсенал.</Russian>
             <Japanese>Zeus Ehnhanced により優先された武器庫を選択します。\nACE3 武器庫が有効化されていない場合、ゲーム標準のバーチャル武器庫へ戻されます。</Japanese>
+            <Korean>ZEN에서 쓰일 아스널을 선택합니다. \n만약 ACE 아스널이 설치되어있지 않다면, BI 아스널로 변경될 것입니다.</Korean>
```
Comment (neilzar) [inline addons/common/stringtable.xml:515]: Please use `Zeus Enhanced` here rather than ZEN.

Comment (astralkr) [reply addons/common/stringtable.xml]: OK. I fixed it.

File: addons/modules/stringtable.xml [hunk 2]
```diff
@@ -1252,120 +1374,140 @@
...
             <Polish>Pochmurność</Polish>
             <Japanese>雲の量</Japanese>
+            <Korean>흐림</Korean>
         </Key>
         <Key ID="STR_ZEN_Modules_ModuleWeather_Overcast_Tooltip">
             <English>Sets the amount of cloud cover, higher values result in a greater likelihood of rain/storms and faster wind speeds.</English>
             <German>Legt die Bedeckung der Wolken fest, höhere Werte führen zu einer höheren Wahrscheinlichkeit von Regen/Stürmen und schnelleren Windgeschwindigkeiten.</German>
             <Polish>Ustala pochmurność, wysoka wartość powoduje większe prawdopodobieństwo deszczu/burzy i szybszego wiatru.</Polish>
             <Japanese>雲が覆う量を決定します。高い値では大雨/嵐のようになり、強い風が吹きます。</Japanese>
+            <Korean>ADAADADAD</Korean>
```
Comment (CreepPork) [inline addons/modules/stringtable.xml:1503]: What's this?

Comment (astralkr) [reply addons/modules/stringtable.xml:1503]: I'm sorry for my miss. I am slept at that time. I will fix it ASAP

Comment (astralkr) [reply addons/modules/stringtable.xml]: Fixed.

**Comment by astralkr** [comment]
SOMEONE PLS REVIEW THIS REQUEST.. T.T I'm sad

**Comment by astralkr** [comment]
LET'S MERGEEEE

**Comment by neilzar** [comment]
> LET'S MERGEEEE

Please try to calm down, none of us speak korean and we would like someone who does to check this first. Yelling and spamming for this to get merged will only annoy us and won't make us do anything. You also need to keep in mind that this isn't our job but voluntary work, all of us have other obligations which come before this mod.

**Comment by astralkr** [comment]
> > LET'S MERGEEEE
> 
> Please try to calm down, none of us speak korean and we would like someone who does to check this first. Yelling and spamming for this to get merged will only annoy us and won't make us do anything. You also need to keep in mind that this isn't our job but voluntary work, all of us have other obligations which come before this mod.
I'm sorry for if you feel my comment as yelling and spamming. I know this isn't your job. I'm really sorry.

**Comment by mharis001** [comment]
Thank you for translating.

**Pull request merged**